docs/graphql/manual/guides/deployment/index.rst

```diff
@@ -13,6 +13,7 @@ Guides: Deployment
 - :doc:`Digital Ocean One-click App on Marketplace <digital-ocean-one-click>`
 - :doc:`Azure Container Instances with Postgres <azure-container-instances-postgres>`
 - :doc:`Google Cloud Platform with Kubernetes engine and Cloud SQL <google-kubernetes-engine-cloud-sql>`
+- :doc:`Render <render>`
```

Comment:
Can we have the full title here, like the others.

Reply:
Yes, except I'm not sure what I would add to it. I suppose we could say "Render with Postgres" or "Render Web Services with Postgres" but the Render product suite is simple enough that the qualification may not be necessary. There is only one way to do this on Render. @anurag May have suggestions here.

Reply:
How about `Render Single Click Deploy with PostgreSQL`